apps/installer/includes/functions.sh

```diff
@@ -9,7 +9,7 @@ function inst_configureOS() {
                 DISTRO=$OSDISTRO
             # If available, use LSB to identify distribution
             elif [ -f /etc/lsb-release -o -d /etc/lsb-release.d ]; then
```

Comment:
```suggestion
            elif command -v lsb_release >/dev/null 2>&1 ; then
```

Reply:
I did some testing and this method works on ubuntu but not debian. changing it to check for the command instead of the file or directory solves this. This method works for both ubuntu and debian.

Reply:
Yeah, that definitely the more elegant solution.